src/app/common/transactions/bitcoin/utils.ts

```diff
@@ -22,8 +22,7 @@ export const getColorFromBitcoinTx = (tx: BitcoinTx) => {
     pending: 'warning.label',
     success: 'stacks',
   };
-
-  return colorMap[statusFromBitcoinTx(tx)] ?? 'feedback-error';
+  return colorMap[statusFromBitcoinTx(tx)] ?? 'error.label';
```

Comment:
unrelated but `type BtcStatusColorMap = Record<BtcTxStatus, PandaTokenType>;`

could tighten types

Reply:
I just noticed this so I fixed it, but we should prob refactor to not use a function to get the color w/ panda ...kinda surprised it is working so will check it.